demos/interpretability/gcn/hateful-twitters-interpretability.ipynb

```diff
@@ -1008,7 +1273,17 @@
    "cell_type": "code",
    "execution_count": 41,
    "metadata": {},
-   "outputs": [],
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "\n",
+      "To change all layers to have dtype float64 by default, call `tf.keras.backend.set_floatx('float64')`. To change just this layer, pass dtype='float64' to the layer constructor. If you are the author of this layer, you can disable autocasting by passing autocast=False to the base Layer constructor.\n",
```

Comment:
Is this some kind of warning output that has now appeared? Or is it something that's always been around and needs to be removed as part of notebook formatting?

Reply:
Nice catch - this is a new tensorflow warning so it has just appeared but not from any of the changes in this PR.

Reply:
I'll manually scrub it for now